src/connectors/imago.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.now-play';
+
+Connector.artistTrackSelector = '#current-track';
+
+Connector.isPlaying = () => $('.play-icon.stop').length === 0;
+
+Connector.pauseButtonSelector = '.play-icon.stop';
```

Comment:
The same thing for play/pause button selectors and the `isPlaying` function here.

`Connector.pauseButtonSelector` is used in default `Connector.isPlaying` function, which is overriden here.

Using `Connector.pauseButtonSelector` only is enough.